please fix the compiling error

```
/home/travis/build/cocos2d/cocos2d-x/cocos/scripting/lua-bindings/manual/3d/lua_cocos2dx_3d_manual.cpp:2048:38: error: ‘obb’ was not declared in this scope
         bool ret = self->intersects(*obb, &distance);
```

--- a/cocos/scripting/lua-bindings/manual/3d/lua_cocos2dx_3d_manual.cpp
+++ b/cocos/scripting/lua-bindings/manual/3d/lua_cocos2dx_3d_manual.cpp
@@ -2044,15 +2044,17 @@ int lua_cocos2dx_3d_Ray_intersects(lua_State* L)
         if (!ok)
             return 0;
 
-        bool ret = self->intersects(*arg0);
+        float distance;
+        bool ret = self->intersects(*arg0, &distance);
         tolua_pushboolean(L, ret);
-        return 1;
+        tolua_pushnumber(L, (lua_Number)distance);
+        return 2;
     }
     luaL_error(L, "%s has wrong number of arguments: %d, was expecting %d \n",  "cc.Ray:intersects",argc, 1);
     return 0;
 #if COCOS2D_DEBUG >= 1
 tolua_lerror:
-    tolua_error(L,"#ferror in function 'lua_cocos2dx_3d_get_OBB_center'.",&tolua_err);
+    tolua_error(L,"#ferror in function 'lua_cocos2dx_3d_Ray_intersects'.",&tolua_err);
     return 0;
 #endif
 }

